pkg/xds/envoy/listeners/v3/rbac_configurer.go

```diff
@@ -0,0 +1,186 @@
+package v3
+
+import (
+	"fmt"
+
+	envoy_listener "github.com/envoyproxy/go-control-plane/envoy/config/listener/v3"
+	rbac_config "github.com/envoyproxy/go-control-plane/envoy/config/rbac/v3"
+	rbac "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/rbac/v3"
+	matcherv3 "github.com/envoyproxy/go-control-plane/envoy/type/matcher/v3"
+
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
+	core_xds "github.com/kumahq/kuma/pkg/core/xds"
+	policies_api "github.com/kumahq/kuma/pkg/plugins/policies/meshtrafficpermission/api/v1alpha1"
```

Comment:
The fact that it has a dependency on `policies_api` is a bit weird. It makes it usable only with MeshTrafficPermission.

I'm ok with merging it as is, but as we go further with the following policies we need to think if this is expected or configurer should be placed in the plugin itself.

Reply:
That's a good point. @michaelbeaumont ended up putting the configurer in the plugins package. Maybe it's a cleaner approach?

Reply:
If it's private to the package it also avoids an IMO unnecessary separate set of tests and we can solely test `Apply`.

Reply:
I agree to have this under `plugins/policies/meshtrafficpermission`, but I think it should be placed out of the `v1alpha1` package. Because Configures have their own versioning which depends on Envoy,  it's `v3` in our case. So it should probably look like:

policies/meshtrafficpermission:
* api
* k8s
* plugin
  * v1alpha1
* xds
  * v3

Reply:
moved under `xds/v3`

Reply:
upd: moved under `xds`